### ISSUE TITLE
Incorrect cursor position when mask configured as blocks

### ISSUE THREAD

**Issue by ElioLooiz** [uNmAnNeR/imaskjs#577]
**Describe the bug**
Incorrect cursor position after focusing an input field when the mask is setted as a number of blocks with custom lazy config

**To Reproduce**
https://codepen.io/eliolooiz-the-looper/pen/jOLXKBL
Focus input field

**Expected behavior**
Cursor position expected be after the first CODE block (after +7) because it is set as static part of the mask and configured in block as {lazy: false}

**Environment:**
 - OS: Windows
 - Browser Chrome
 - Version 95.0.4638.69
 - IMask version 6.2.2

**Additional context**
If the CODE part will provided as pattern in mask prop it works partly correct, but sometimes cursor will get incorrect position (for example after Ctrl+X after all text selection).

**Comment by VechniiNewbie** [comment]
you makes first mask not fixed by doing it only in block
i think this should work
```js
const masksOptions = {
  phone: {
    mask: '{+7} NUM',
    lazy: false,
    blocks: {
      NUM: {
        mask: '(000) 000-0000',
        lazy: true
      }
    }
  }
};
```

**Comment by ElioLooiz** [comment]
> you make first block not "fixed" by doing it only in block i think this should work
> 
> ```js
> const masksOptions = {
>   phone: {
>     mask: '{+7} NUM',
>     lazy: false,
>     blocks: {
>       NUM: {
>         mask: '(000) 000-0000',
>         lazy: true
>       }
>     }
>   }
> };
> ```

Yep, it works in some cases, but if you enter a number, then select all -> cut it -> blur from input -> focus again -> cursor has incorrect position again (before CODE)

So I think the problem really exists, because in my understanding, the cursor should move to the first character available for input, and not to a static one. As it works when we provide mask as simple pattern like '{+7} (000) 000-0000'

**Comment by uNmAnNeR** [comment]
fixed in #606
